firedrake/ml/pytorch/pytorch_custom_operator.py

```diff
@@ -0,0 +1,92 @@
+import torch
+import collections
+from functools import partial
+
+from firedrake.ml import load_backend
+from firedrake.function import Function
+from firedrake_citations import Citations
+
+from pyadjoint.reduced_functional import ReducedFunctional
+
+
+Citations().add("Bouziani2023", """
+@inproceedings{Bouziani2023,
+ title = {Physics-driven machine learning models coupling {PyTorch} and {Firedrake}},
+ author = {Bouziani, Nacime and Ham, David A.},
+ booktitle = {{ICLR} 2023 {Workshop} on {Physics} for {Machine} {Learning}},
+ year = {2023},
+ doi = {10.48550/arXiv.2303.06871}
+}
+""")
+
+
+backend = load_backend("pytorch")
```

Comment:
If you use free functions this could simply be (I think):
```py
from firedrake.ml.pytorch import backend
```

Reply:
I think I don't understand what you mean.

Reply:
If one uses free functions (i.e. not methods in a class) then accessing `from_ml_backend` etc could simply be an import, instead of via a special `load_backend` function.

This is nitpicking though. My main query is about whether or not one would want >1 of `PytorchBackend`.